format/index.md

```diff
@@ -382,6 +387,10 @@ responses are called "compound documents".
 In a compound document, all included resources **MUST** be represented as an
 array of resource objects in a top-level `"included"` member.
 
+In a compound document, any resource which actually references other resources included 
+into the same document (either in top-level `"data"` or `"included"` members)
+**MUST** include references to those resources in its own `"links"` member.
+
```

Comment:
I believe this is already stated under [Resource Relationships](http://jsonapi.org/format/#document-structure-resource-relationships):

> If a link object refers to resource objects included in the same compound document, it **MUST** include resource linkage to those resource objects.